tests/sqlparser_common.rs

```diff
@@ -3495,7 +3495,7 @@ fn parse_create_table_on_cluster() {
     let sql = "CREATE TABLE t ON CLUSTER '{cluster}' (a INT, b INT)";
     match generic.verified_stmt(sql) {
         Statement::CreateTable(CreateTable { on_cluster, .. }) => {
-            assert_eq!(on_cluster.unwrap(), "{cluster}".to_string());
+            assert_eq!(on_cluster.unwrap(), "'{cluster}'".to_string());
```

Comment:
hmm as far as I can tell the new behavior looks more reasonable to me, curious though looking at the previous behavior, shouldn't this have been wrapped around the single quotes already? (wondering if I'm missing something, because this line seems to suggest that this `replace` being done in the display wasnt working? `on_cluster.replace('{', "'{").replace('}', "}'")`

Reply:
@iffyio For the original implementation[1], it changed the single quote string into String, so it use the replace way for the special char like '{' which must be inside the the single quote string.

[1] https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/parser/mod.rs#L5390

Reply:
Ah right that makes sense!